src/lib/plugin_apis/fs.api

```diff
@@ -204,6 +205,36 @@ GType bd_fs_vfat_info_get_type () {
  */
 gboolean bd_fs_wipe (const gchar *device, gboolean all, GError **error);
 
+/**
+ * bd_fs_unmount:
+ * @spec: mount point or device to unmount
+ * @lazy: enable/disable lazy unmount
+ * @force: enable/disable force unmount
+ * @extra: (allow-none) (array zero-terminated=1): extra options for the unmount
+ *                                                 currently only 'run_as_uid'
+ *                                                 and 'run_as_gid' is supported
```

Comment:
Please also document how the value (``key``) should look like (and that it cannot be ``"0"``).